components/core/LoginButton/Login.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <BaseButtonLink
```

Comment:
As I said above probably CoreLoginButton doesn't sound correct since we will probably only use a LoginButton once, so probably need to change the file architecture fore those

Reply:
I feel like there is a distinction between `base` and `core`, but I don't know how to explain it 😬 . We can just use one or the other though, if you think that makes more sense and is less confusing.

Reply:
Imo, core is more of a core component library that is imported, whereas base is more inheritance focused? When I hear base I think of "base class".

Reply:
I think that's exactly what I was feeling, just didn't know how to articulate it. @legrandgrand what do you think about the distinction now?

Reply:
Sounds good to me then, we can use that if there is a difference